src/index.ts

```diff
@@ -4,6 +4,9 @@ export * from 'near-runner';
 
 export declare interface Runner {
   test(description: string, fn: RunnerFn): void;
+  networkIsTestnet(): boolean;
+  networkIsSandbox(): boolean;
+  getNetworkFromEnv(): 'sandbox' | 'testnet';
```

Comment:
Can we get this `Runner` to extend the one from `near-runner-js`? We shouldn't need to copy-paste all of these.